marklogic-data-hub/src/main/java/com/marklogic/hub/mapping/MappingImpl.java

```diff
@@ -61,7 +61,7 @@ public MappingImpl(String name, HubEntity entity) {
         if (entity != null) {
             InfoType info = entity.getInfo();
             if (info != null) {
-                String prefix = info.getBaseUri() != null ? info.getBaseUri() + "/" : "";
+                String prefix = info.getBaseUri() != null ? info.getBaseUri() : "";
```

Comment:
Pinging @ryanjdew  on this - why the change here? I believe "/" was included to ensure that the base URI ends with a "/" which I think is a requirement of ES. Perhaps the "/" should only be included if info.getBaseUri() doesn't end with "/"?

Reply:
This just reverts the changes made in https://github.com/marklogic/marklogic-data-hub/pull/4065 since we do need a baseUri to end with `/`

Reply:
@rjrudin Entity Services will add the slash ("/") appropriately whether or not the base URI ends with a slash.